subsys/bluetooth/host/l2cap.c

```diff
@@ -2028,37 +2038,66 @@ static int l2cap_chan_le_send(struct bt_l2cap_le_chan *ch,
 	return len;
 }
 
+/* return next netbuf fragment if present, also copy metadata */
```

Comment:
```suggestion
/* return next netbuf fragment if present, also assign metadata */
```

A pedantic request from me, but I think `copy` is a bit wrong to use here when it is specifically not copied :)